src/cli.js

```diff
@@ -26,6 +26,7 @@ const cli = meow(
         --configFile, -c File path of .npmpackagejsonlintrc.json
         --ignorePath, -i Path to a file containing patterns that describe files to ignore. The path can be absolute or relative to process.cwd(). By default, npm-package-json-lint looks for .npmpackagejsonlintignore in process.cwd().
         --maxWarnings, -mw Maximum number of warnings that can be detected before an error is thrown.
+        --allowEmptyTargets Do not throw an error when a list of targets is empty.
```

Comment:
Would you mind changing `Targets` to `Patterns` so it matches the other docs and error messages?

```suggestion
        --allowEmptyPatterns Do not throw an error when the list of patterns is empty.
```